docs/app/Examples/addons/Confirm/Variations/ConfirmExampleSize.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import { Button, Confirm } from 'semantic-ui-react'
+
+class ConfirmExampleSize extends Component {
+  state = { open: false }
+
+  show = () => this.setState({ open: true })
+  handleConfirm = () => this.setState({ open: false })
+  handleCancel = () => this.setState({ open: false })
+
+  render() {
+    return (
+      <div>
+        <Button onClick={this.show}>Show Large</Button>
+        <Confirm
+          open={this.state.open}
+          size="large"
```

Comment:
let's keep consistency with single quotes.

Reply:
Hi, @layershifter. I noticed that there is another file in this same path called `ConfirmExampleButtons`updated a year ago. This file I mention also mixes single quotes and double quotes, I remember that there are some lint tests for this, could this be indicating that these files are not in the coverage of the lint test? or maybe I'm wrong and there is no test for this, if that's the case I think we should create one, what do you think?.